### PR TITLE
feat: updated the ape core and vyper

### PR DESCRIPTION
**Pull request by Ninjagod1251**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -138,7 +139,7 @@ async def get_task_status(task_id: str) -> TaskStatus:
 
 
 @app.get("/exceptions/{task_id}")
-async def get_task_exceptions(task_id: str) -> list[str]:
+async def get_task_exceptions(task_id: str) -> dict:
```
Comment (antazoey) [inline main.py:142]: We may want to add return types to the other tasks as well, I noticed most of them don't have them.

**Review by antazoey — APPROVED** [review]
looks good
(one nit, can be for future)

**Review by fubuloubu — APPROVED** [review]
Do some local testing with the docker container, then we can merge and release this

**Comment by Ninjagod1251** [comment]
If the return types are correct, it is ready to merge @antazoey

File: main.py [hunk 2]
```diff
@@ -155,7 +156,7 @@ async def get_task_exceptions(task_id: str) -> list[str]:
 # NOTE: `response_model=None` so that we only use our own validation
 #   from ethpm_types.
 @app.get("/artifacts/{task_id}", response_model=None)
-async def get_compiled_artifact(task_id: str):
+async def get_compiled_artifact(task_id: str)-> dict:
```
Comment (fubuloubu) [inline main.py:159]: This actually returns a manifest

**Review by fubuloubu — APPROVED** [review]

**Pull request merged**